helmfile/values/kube-prometheus-stack-sc.yaml.gotmpl

```diff
@@ -29,7 +29,7 @@ alertmanager:
   config:
     # See https://prometheus.io/docs/alerting/configuration/
     route:
-      group_by: [cluster, alertname, severity]
+      group_by: '{{ toYaml .Values.user.alertmanager.group_by | nindent 2}}'
```

Comment:
Why the `nindent`?

Or does it produce:
```yaml
group_by:
  - cluster
  - alertname
  - severity
```

If so, shouldn't it be `nindent 4` since this is 4 indents in?

Reply:
Exactly like that, yes!